website/addons/base/views.py

```diff
@@ -203,6 +204,21 @@ def create_waterbutler_log(payload, **kwargs):
     node = kwargs['node'] or kwargs['project']
 
     if action in (NodeLog.FILE_MOVED, NodeLog.FILE_COPIED):
+        dest = payload['destination']
```

Comment:
This block should be after line 225, 222-225 is validating that all the correct fields exist.